hieradata/common.yaml

```diff
@@ -380,6 +380,9 @@ govuk::apps::specialist_publisher::enabled: true
 govuk::apps::specialist_publisher::nagios_memory_warning: 800
 govuk::apps::specialist_publisher::nagios_memory_critical: 900
 
+govuk::apps::specialist_publisher_rebuild::enabled: true
+govuk::apps::specialist_publisher_rebuild::publish_pre_production_finders: true
```

Comment:
Good spot. 🔬

We've updated this pull request to only hoist the 'enabled' setting up.

Thanks